content/docs/0.6.0/reference/monitoring/dynatrace/index.md

```diff
@@ -161,22 +161,31 @@ During the evaluation of a quality gate, the Dynatrace SLI provider is required
   kubectl apply -f service.yaml
   ```
 
-**Verify deployment in your cluster:**
-
- Execute the following commands to verify the deployment of the Dynatrace SLI provider:
+* To verify that the deployment has worked, execute:
 
   ```console
   kubectl get pods -n keptn | grep dynatrace-sli
   ```
 
+---
+
+**Provider configuration:**
+
+To tell the *dynatrace-sli-service* how to acquire the values of an SLI, the correct query needs to be configured. This is done by adding an SLI configuration to a project, stage, or service using the [add-resource](../../cli/#keptn-add-resource) command. The resource identifier must be `dynatrace/sli.yaml`.
+
+* In the below example, the SLI configuration as specified in the `sli-config-dynatrace.yaml` file is added to the service `carts` in stage `hardening` from project `sockshop`. 
+
   ```console
-  dynatrace-sli-service
-  dynatrace-sli-service-monitoring-configure-distributor
+  keptn add-resource --project=sockshop --stage=hardening --service=carts --resource=sli-config-dynatrace.yaml --resourceUri=dynatrace/sli.yaml
   ```
 
+**Note:** The add-resource command can be used to store a configuration on project-, stage-, or service-level. The overwriting strategy of Keptn is from the most generic to the more specific entity, meaning that a configuration on service-level overrides a configuration on stage-level, which overrides a configuration on the project-level. If, for example, an SLI is configured on service-level and on project-level, the configuration on service level overrules the other. 
```

Comment:
Streamline: "service level" should be "service-level"

Reply:
I think the term "overwriting strategy of Keptn" is (too) complicated.
Can we replace 
"The overwriting strategy of Keptn..." 
to
"Keptn always first uses SLIs contained in the service-level, then Keptn uses SLIs contained in the stage-level, and finally Keptn uses SLIs contained in the project-level."

Reply:
I changed the wording and removed complexity.